apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -61,6 +62,8 @@ public void Dispose()
 
         private async Task EnsureTestDatabaseImplAsync()
         {
+            TestLogger.Install();
+            await Task.Delay(1); // this line exists so the others can be commented.
```

Comment:
when I use my scratch database, I usually comment out all the creation bits and dispose.
1 millisecond isn't a big deal, but sure.